apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,45 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Menuconfig Setup
+**For build configuration details refer to**
+'build/configs/rtl8721csm/security_hal_test_tz/defconfig'
+
+**Enable web server**
+> Networking support > Protocols > Webserver 
+
+> Application Configuration > Examples > Webserver example
+
+> Application Configuration > Examples > Webserver test
+
+## webserver_test usage :
+
+**webserver_test iterations method uri [options...]**
+
+- method and url parameters should be given,  [options] parameters are optional.
+- method is one if "GET", "POST", "PUT", "DELETE".  
```

Comment:
method -> possible values "GET"/"POST"/"PUT"/"DELETE"
OR 
in your line chg 'if' to 'of'